firefox-ios/firefox-ios-tests/Tests/XCUITests/HomePageSettingsUITest.swift

```diff
@@ -404,17 +404,17 @@ class HomePageSettingsUITests: BaseTestCase {
         addWebsitesToShortcut(website: "www.bestbuy.com")
         addWebsitesToShortcut(website: "www.instagram.com")
         if !iPad() {
-            validateNumberOfTopSitesDisplayed(row: 0, topSites: 4)
-            validateNumberOfTopSitesDisplayed(row: 1, topSites: 8)
-            validateNumberOfTopSitesDisplayed(row: 2, topSites: 12)
-            validateNumberOfTopSitesDisplayed(row: 3, topSites: 13)
+            validateNumberOfTopSitesDisplayed(row: 0, minBoundary: 1, maxBoundary: 5)
```

Comment:
Can we find the element using `element(boundBy: x)` or similar? using boundaries like those could change from devices

Reply:
I added this solution because there are differences between what i have locally and what results are on jenkins, so to make sure the visible top sites are displayed based on the shortcut setting, i added these ranges for every row.
Adding a single element could cause the test to fail because of these differences.